source/issues-and-help.txt

```diff
@@ -0,0 +1,50 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant MongoDB C#/.NET community
+that includes users with varying levels of experience using
+the driver. We find the quickest way to get support for general
+questions is through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
+
+To learn more, refer to our :manual:`support channels </support>`.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the C#/.NET driver,
+please open a case in our issue management tool, JIRA:
+
+* `Create an account and login <https://jira.mongodb.org>`_.
+* Navigate to `the .NET Driver project <https://jira.mongodb.org/browse/CSHARP>`_.
+* Click **Create Issue**. Please provide as much information as possible about the
+  issue and the steps to reproduce it.
+
+Bug reports in JIRA for the C#/.NET driver and the Core Server (i.e. SERVER) project are **public**.
+
+If you’ve identified a security vulnerability in a driver or any other
+MongoDB project, please report it according to the instructions found in
+the :manual:`Create a Vulnerability Report </tutorial/create-a-vulnerability-report>`.
+
+Pull Requests
+-------------
+
+We are happy to accept contributions to help improve the driver. We will
+guide user contributions to ensure they meet the standards of the
+codebase. Please ensure that any pull requests include documentation,
+tests, and pass the existing test suite.
```

Comment:
Question: I wasn't able to find the existing test suite mentioned here. The Contributing.md file linked in this section only mentions writing your own tests. Are there directions we can link to in order to pass the existing test suite? Or should this description be changed to match the requirements on Contributing.md?

Reply:
I used the wording from the outline document... but yes, it appears that there are no given tests. I updated the language.

Reply:
Did the DBX team provide information on how they expect to receive contributions? If you didn't get that text from them whether directly, if that's not what the DBX team wants to communicate, definitely update whatever is needed (Contributing.md and/or this paragraph) to make it accurate.